src/views/Home.jsx

```diff
@@ -14,11 +22,48 @@ export function Home({ createNewToken, setListToken }) {
 		navigate('/list');
 	};
 
+	const handleTokenChange = (e) => {
+		setExistingToken(e.target.value);
```

Comment:
This fires on every keystroke. That's not the end of the world _right now_ because you are just updating state with no downstream effects, but I would consider a debounce here anyway